docs/standard/assembly/unloadability.md

```diff
@@ -108,9 +108,10 @@ In LLDB:
 plugin load /path/to/libsosplugin.so
 ```
 
-Let's try to debug an example program that has problems with unloading. Source code is included below. When you run it under WinDbg, the program breaks into the debugger right after attempting to check for the unload success. You can then start looking for the culprits.
+Let's debug an example program that has problems with unloading. Source code is included below. When you run it under WinDbg, the program breaks into the debugger right after attempting to check for the unload success. You can then start looking for the culprits.
```

Comment:
It would be ideal to get rid of first-person altogether, but as it appears to be pervasive in the doc it's probably not worth the effort.